osu.Game.Tests/Visual/Background/TestSceneUserDimBackgrounds.cs

```diff
@@ -371,7 +371,7 @@ public TestPlayerLoader(Player player)
 
             public void TriggerOnHover() => OnHover(new HoverEvent(new InputState()));
 
-            public bool IsBlurCorrect() => ((FadeAccessibleBackground)Background).CurrentBlur == new Vector2(BACKGROUND_BLUR);
+            public bool IsBlurCorrect() => ((FadeAccessibleBackground)Background).CurrentBlur == new Vector2(BackgroundBlur);
```

Comment:
This doesn't compile. If the current user settings are to be fetched, resolve an `OsuConfigManager` (examples how to do so can be found in lots of existing test scenes).